torch/onnx/symbolic_opset13.py

```diff
@@ -0,0 +1,132 @@
+from torch.onnx.symbolic_helper import _block_list_in_opset
+import torch
+import torch.onnx.symbolic_helper as sym_help
+from torch.onnx.symbolic_helper import parse_args, _unimplemented
+
+block_listed_operators = ['embedding_bag']
```

Comment:
is it due to loop that embedding_bag not supported?

Reply:
Yes, exactly